test/test_jewish_calendar.py

```diff
@@ -899,3 +900,9 @@ def test_sof_zman_kiddush_levana_between_moldos(self):
         expected_time = first_molad + timedelta(microseconds=expected_offset)
         # round for floating microsecond precision inconsistency
         self.assertEqual(calendar.sof_zman_kiddush_levana_between_moldos().toordinal(), expected_time.toordinal())
+
+    def test_date_arithmetic_returns_inherited_type(self):
+        for instance, type in [(JewishCalendar(5783, 1, 1), JewishCalendar), (JewishDate(5783, 1, 1), JewishDate)]:
```

Comment:
I'd prefer to keep other types out of this batch of tests, JewishCalendar alone should suffice here.  An alternative approach would be to test the method in TestJewishDate, and create a dummy class to ensure the type isn't being lost.